core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/BlockletDataRefNode.java

```diff
@@ -133,6 +134,21 @@
     return null;
   }
 
+  @Override
+  public boolean[] isMinMaxSet() {
+    BlockletIndex blockletIndex =
+        blockInfos.get(index).getDetailInfo().getBlockletInfo().getBlockletIndex();
```

Comment:
No need to pass this information from driver to executor.

Reply:
ok